src/Form/ParticipationType.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Form;
+
+use App\Entity\Participation;
+use App\Repository\ConferenceRepository;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Component\Validator\Constraints\NotNull;
+
+class ParticipationType extends AbstractType
+{
+    public function __construct(
+        private ConferenceRepository $conferenceRepository,
+    ) {
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('conference', DatalistType::class, [
+                'choices' => $this->conferenceRepository
+                    ->getFutureConferencesQueryBuilder()
+                    ->getQuery()
+                    ->execute(),
+                'mapped' => false,
```

Comment:
We, this part seems wrong, cf the comment in the controller. You should attach a data transformer here
(Or create a new `ConferenceDataListType`, and put everything there)